### PR TITLE
CTCP-552: prototype test pack

### PR DESCRIPTION
**Pull request by PeterBandura**

### CODE REVIEW

File: source/documentation/index.md [hunk 1]
```diff
@@ -19,10 +19,11 @@ description: Software developers, designers, product owners or business analysts
 
 This test pack is specifically designed for use with the CTC Traders API.
 
-You must use this with the CTC Traders API using the sandbox environment with the Trader Test Service. It gives you sample scenarios to use when checking your software functions correctly and that it is compatible with HMRC systems. Use it to test all scenarios for arrivals and departure movements to and from both Great Britain (GB) and Northern Ireland.
```
Comment (drnaylor-oc) [inline source/documentation/index.md:22]: GB is still used as an acronym further down - so either this should be reinstated, or the acronym should be expanded below).

(Do note that this is a copy/patse from the Word based test pack - do we want to deviate from that at this stage?)

File: source/documentation/simplified_departures.html.md [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+title: CTC Traders phase 5 test pack
+weight: 10
+description: Software developers, designers, product owners or business analysts. Integrate your software with Common Transit Convention Traders API.
+---
+
+# Simplified departures
+
+## Great Britain to Europe (using Italy as example destination)
+
+### DECLARATION 1: GB to IT – Safety and Security; Simplified Declaration with Missing Place of Loading Code
```
Comment (drnaylor-oc) [inline source/documentation/simplified_departures.html.md:11]: Is the intention for each declaration to be a separate page, or that all simplified departures be on one page, then all simplified arrivals on another etc?

The reason I ask is that if it's the former, then you're basically going to end up with many pages named "Simplified departures". It's probably worth knowing what we want to do early.

(Note that I added a couple of pages in the "simplified" directory (https://github.com/hmrc/ctc-traders-phase5-test-pack/tree/main/source/documentation/simplified) so they're going to show up too)

File: source/documentation/index.md (hunk 1 above)
Comment (drnaylor-oc) [reply source/documentation/index.md:22]: Personally, I don't think it matters either way, it just depends on the scope of what we're doing. Do remember that this is only a prototype, only destined for QA for the time being.

If GB and NI are forbidden (quite curious as to why that is, but that's not really a discussion for here), I guess go with that. It probably doesn't matter at this stage though.

Comment (PeterBandura) [reply source/documentation/index.md:22]: I've attempted to replace "GB" throughout the topic. Tom and Lydia are both offline at the moment. I'll try and find out the reasoning when I get the chance. Both GB and XI are used in the code and I think there are no problems with that - it's just in content where restrictions seem to apply.

Comment (drnaylor-oc) [reply source/documentation/index.md:22]: Sure - it's not important, just a curiosity - don't need to explain it here!

File: source/documentation/simplified_departures.html.md (hunk 2 above)
Comment (PeterBandura) [reply source/documentation/simplified_departures.html.md:11]: Sorry Dan I didn't realise you were working in this repo. Looks like I've been treading on your toes here. The approach I thought I'd take was the latter: to create a page for each of the following to match the top level in the Word doc:
- Simplified departures
- Normal departures
- Simplified arrivals
- Normal arrivals
- and so on.
and then to cram everything within the Simplified departures section in the Word doc into the "Simplified departures" page and so on.

**Pull request closed without merging**